sql/src/influxql/mod.rs

```diff
@@ -19,11 +19,22 @@ pub mod error {
         ))]
         BuildSchema { msg: String, backtrace: Backtrace },
 
-        #[snafu(display("Failed to build influxql plan, msg:{}, err:{}", msg, source))]
-        BuildPlan { msg: String, source: GenericError },
+        #[snafu(display(
+            "Failed to build influxql plan with cause, msg:{}, err:{}",
+            msg,
+            source
+        ))]
+        BuildPlanWithCause { msg: String, source: GenericError },
+
+        #[snafu(display(
+            "Failed to build influxql plan with no cause, msg:{}.\nBacktrace:{}",
```

Comment:
It's best we can add a macro to fix this. I always forge this.

```
display_backtrace!("Failed to build influxql plan with no cause, msg:{}.")
```
It will rewrite to line above.